napari/benchmarks/benchmark_tracks_layer.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+
+from napari.layers import Tracks
+
+
+class TracksSuite:
+    param_names = ['size', 'n_tracks']
+    params = [(5 * np.power(10, np.arange(2, 7))).tolist(), [10, 100, 1000]]
+
+    def setup(self, size, n_tracks):
+        """
+        Create tracks data
+        """
+        if 10 * n_tracks > size:
+            # number of tracks are adjusted if not 10 times larger than size
+            n_tracks = size // 10
+
+        rng = np.random.default_rng()
```

Comment:
Based on [the numpy docs](https://numpy.org/doc/stable/reference/random/generator.html#numpy.random.default_rng), this call will generate unpredictable random numbers. I think we should seed this to make things predictable.

```suggestion
        rng = np.random.default_rng(0)
```

Reply:
Ok

Reply:
Done